test/check-application

```diff
@@ -302,6 +304,22 @@ WantedBy=multi-user.target default.target
         self.execute(auth, f"{systemctl_cmd} daemon-reload; {systemctl_cmd} start {name}.service")
         self.execute(auth, f"until {systemctl_cmd} is-active {name}.service; do sleep 1; done")
 
+    def createQuadletPod(self, name: str, *, auth: bool = False) -> None:
+        quadlet = f"""
+[Pod]
+PodName={name}
+"""
+        service_file = f"/etc/containers/systemd/{name}.pod"
+        if not auth:  # user session
+            service_file = f"/home/admin/.config/containers/systemd/{name}.pod"
+
+        systemctl_cmd = "systemctl" if auth else "systemctl --user"
+        self.addCleanup(self.execute, auth, f"{systemctl_cmd} daemon-reload")
+        self.write_file(service_file, quadlet)
+        self.addCleanup(self.execute, auth, f"{systemctl_cmd} stop {name}-pod.service")
+        self.execute(auth, f"{systemctl_cmd} daemon-reload; {systemctl_cmd} start {name}-pod.service")
+        self.execute(auth, f"until {systemctl_cmd} is-active {name}-pod.service; do sleep 1; done")
```

Comment:
I wonder, is that `until` loop really necessary? systemctl normally blocks `start` until the unit is either active (and then succeeds) or failed (and then exits nonzero), unless you specify `--no-block`. So this ought to be dead code.

Reply:
Dropped (it was introduced earlier than this PR already, so separate commit)